src/bebone/gfx/opengl/opengl_shader_factory.cpp

```diff
@@ -19,17 +19,17 @@ namespace bebone::gfx {
         return false;
     }
 
-    GLShader GLShaderFactory::create_shader(const std::string& path, const ShaderType& shader_type, const GLShaderProperties& properties) {
+    GLShader GLShaderFactory::create_shader(const std::string& path, const ShaderType& type, const GLShaderProperties& properties) {
```

Comment:
If you changed the argument name in cpp file, then also don't forget to rename it in [header file](https://github.com/Maksasj/bebone/blob/3b7d268806f355a4c4bba452da3c2bd2a487edf6/src/bebone/gfx/opengl/opengl_shader_factory.h#L29C56-L29C58) 😁

Reply:
![Uploading image.png…]()